Source/Core/RequestState.js

```diff
@@ -0,0 +1,21 @@
+/*global define*/
+define([
+        '../Core/freezeObject'
+], function(
+        freezeObject) {
+    'use strict';
+
+    /**
+     * @private
```

Comment:
I mentioned this already, but I'm not convinced we can get away with making these private.  Can someone implement their own TerrainProvider or ImageryProvider without using these objects?